src/service/utils/viewConnection.js

```diff
@@ -149,6 +153,7 @@ const setStream = async (entrySrcObject) => {
     const opositeElementRef =
       state.Controls.currentElementRef === 'player' ? 'player2' : 'player'
     const mediaTag = document.getElementById(opositeElementRef)
+    const drmAudio = document.getElementById('drm-audio-' + opositeElementRef)
```

Comment:
nitpick: (non-blocking) @gregdolby I know it wasn't your typo, but opposite is misspelled